client/src/components/LogIn.js

```diff
@@ -0,0 +1,84 @@
+import { SiDatadog } from "react-icons/si";
+import { Link } from "react-router-dom";
+import { useHistory } from "react-router-dom";
+
+import "./LogIn.css";
+const baseUrl = "http://localhost:4000/login";
+
+export default function Cover() {
+  let history = useHistory();
+
+  function handleLogInSubmit(e) {
+    e.preventDefault();
+    const initDetails = {
+      method: "post",
+      headers: {
+        "Content-Type": "application/json; charset=utf-8",
+      },
+      mode: "cors",
+      body: JSON.stringify({
+        username: e.target[0].value,
+        password: e.target[1].value,
+      }),
+    };
+    fetch(baseUrl, initDetails)
+      .then((res) => {
+        if (res.status !== 200) {
+          console.log("There was an error, please check again.");
```

Comment:
you can turn this into a prompt, so the user also knows. Just in case they are not checking the console ;)